### PR TITLE
New version: JetBrains.IntelliJIDEA.Ultimate.EAP version 223.7401.7

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | IntelliJ IDEA Ultimate Edition (EAP) | IntelliJ IDEA Community Edition 223.7401.7    |
| Version     | 223.7401.7     | 223.7401.7 |
| Publisher   | JetBrains s.r.o.   | JetBrains s.r.o.      |
| ProductCode |           | IntelliJ IDEA Community Edition 223.7401.7    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87451)

### CODE REVIEW

File: manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/223.7401.7/JetBrains.IntelliJIDEA.Ultimate.EAP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2022-11-03
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download-cdn.jetbrains.com/idea/ideaIU-223.7401.7.exe
+  InstallerSha256: EAED433C5513C3F302A6C1F7EB8D86724BFF8B1F8A0F8A4D99D9F9620B687FE1
+  ProductCode: IntelliJ IDEA 223.7255.1
```
Comment (mdanish-kh) [inline manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/223.7401.7/JetBrains.IntelliJIDEA.Ultimate.EAP.installer.yaml:20]:
```suggestion
  ProductCode: IntelliJ IDEA 223.7401.7
```

**Comment by mdanish-kh** [comment]
@Trenly @vedantmgoyal2009 - Shouldn't the PackageName be `IntelliJ IDEA` instead of `IntelliJ IDEA Ultimate Edition (EAP)`? I have seen such naming in other JetBrains manifests where the PackageName doesn't quite match the Control Panel so wanted to learn if this is intentional.
image

**Comment by Trenly** [comment]
@jedieaston knows more about it than I do

**Comment by jedieaston** [comment]
> @Trenly @vedantmgoyal2009 - Shouldn't the PackageName be `IntelliJ IDEA` instead of `IntelliJ IDEA Ultimate Edition (EAP)`? I have seen such naming in other JetBrains manifests as well where the PackageName doesn't quite match the Control Panel so wanted to learn if this is intentional. image

PackageName should keep the edition and EAP, IMO. We can use the `AppsAndFeaturesEntries` if we need to ensure a match, but when you're searching through our catalog there wouldn't be a way to tell community from ultimate or EAP from stable otherwise.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**